io/disk/lvsetup.py

```diff
@@ -82,8 +81,7 @@ def setUp(self):
         self.lv_name = lv_name
         if lv_utils.lv_check(vg_name, lv_snapshot_name):
             self.skip('Snapshot %s already exists' % lv_snapshot_name)
-        self.mount_loc = 'avocado_%s' % self.fs_name
-        os.mkdir(self.mount_loc)
+        self.mount_loc = self.srcdir
```

Comment:
Thought it would be better to use `self.srcdir` as uniform across all tests for mounting.